models/review copy.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose')
```

Comment:
Hi @cynthiaeddy! Are there supposed to be two review.js files (review.js and review copy.js)? They seem identical.

Reply:
nope, my bad.